### ISSUE TITLE
Send to many feature

### ISSUE THREAD

**Issue by deplacer** [moezbhatti/qksms#564]
Allow user to select multiple contact when they compose new message. @ #

**Comment by moezbhatti** [comment]
This is already possible if you select the recipient field again and just enter another contact

**Comment by deplacer** [comment]
Yes but the problem is you need to type the name of every contact you want to add. It's time consuming when you send a group message to 10 persons. How about add  a button that allows users to browse to their contact list.
![step_4_group_sms_android_thumb](https://cloud.githubusercontent.com/assets/7690158/17203028/8cdb0d1a-54cf-11e6-84c6-0069d4c66156.png)

**Comment by spocko** [comment]
I agree that it would be nice if there was an easier way to select multiple contacts.

Another related suggestion:

Currently when composing a new message, the full contacts list is displayed. This is simple but limiting. I end up typing the name of the recipient, or doing a lot of scrolling. Most other messenger apps have other selection/filter options, such as favorites, recent, frequent, existing conversations, groups, etc. I'm not suggesting that all of those should be added to QKSMS, but something more than just the full contacts list would make it easier to select one or multiple contacts.
